utam-core/src/main/java/utam/core/element/Element.java

```diff
@@ -86,4 +185,40 @@
     LEFT,
     RIGHT
   }
+
+  /**
+   * options for a drag and drop actions: element, offset and duration.
+   *
+   * @since 236
+   */
+  interface DragAndDropOptions {
+
+    /**
+     * Duration of hold
+     *
+     * @return null by default
+     */
+    default Duration getHoldDuration() {
+      return null;
+    }
+
+    /**
+     * Target position for drop, as an element
+     *
+     * @return null by default
+     */
+    default Element getTargetElement() {
+      return null;
+    }
+
+    /**
+     * Target position for drop, as offset coordinates
+     *
+     * @return null by default
+     */
+    default Point getOffset() {
```

Comment:
@jimevanssfdc 
is it ok to use Java.Point here? we don't want to use Selenium Point as this is Selenium agnostic wrapper for WebElement

Reply:
Let's create our own data structure. AWT may bring other baggage that we don't really want to have to deal with going forward. Or, if everything is just going off of an Options class anyway, do we really need it?

Reply:
added to method getXOffset and getYOffset and moved this interface outside